shared/timing/timing_test.go

```
@@ -0,0 +1,159 @@
+	if carrier != nil {
+		t.Errorf("Inject with no active span = %v, want nil", carrier)
+	}
+	ctx := Extract(context.Background(), map[string]string{"traceparent": "00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-01"})
+	if ctx == nil {
+		t.Error("Extract returned nil context")
+	}
+}
+
+func TestInitTracingNoopWhenUnset(t *testing.T) {
+	t.Setenv("OTEL_EXPORTER_OTLP_ENDPOINT", "")
+	os.Unsetenv("OTEL_EXPORTER_OTLP_ENDPOINT")
```

golangci-lint (2.12.2)

[error] Error return value of `os.Unsetenv` is not checked

(errcheck)